tasks/managed/rh-sign-image/rh-sign-image.yaml

```diff
@@ -115,7 +121,7 @@ spec:
         if [ "${REQUESTTYPE}" == "internal-pipelinerun" ] ; then
           requestType=internal-pipelinerun
           EXTRA_ARGS=(
-          --service-account "${service_account_name}"
+           --service-account "${service_account_name}"
```

Comment:
This extra space seems like a mistake? If it's intended and you want to visually indent this, then I'd recommend two spaces.

Reply:
fixed